identity/app/views/profile/deletion/accountDeletionBlock.scala.html

```diff
@@ -0,0 +1,84 @@
+@import idapiclient.responses.Error
+
+@(
+    page: model.Page,
+    idRequest: services.IdentityRequest,
+    idUrlBuilder: services.IdentityUrlBuilder,
+    errors: List[Error],
+    user: com.gu.identity.model.User,
+    userContentAccess: services.ContentAccess
+)(implicit
+    request: RequestHeader,
+    messages: play.api.i18n.Messages,
+    context: model.ApplicationContext,
+)
+
+@import views.html.fragments.registrationFooter
+
+@emailIsValidated = @{
+    user.statusFields.userEmailValidated.fold(false)(identity)
+}
+
+
+<div class="identity-wrapper monocolumn-wrapper">
+
+    @if(emailIsValidated) {
+        <div class="identity-section">
+            <p class="identity-section__text">
+                We've noticed you have an active membership, recurring contribution or subscription with the Guardian, so you won't be able to automatically delete your account online. Please follow the instructions below first and try again if you would still like delete your account.
+            </p>
+
+            </br>
```

Comment:
Can we not achieve this spacing without having to resort to `<br/>`s as they are [not semantic](https://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br)